www/src/data/sidebars/doc-links.yaml

```
@@ -289,6 +289,8 @@
       link: /docs/gatsby-config/
     - title: Gatsby Node APIs
       link: /docs/node-apis/
+    - title: Gatsby Node Helpers
+      link: /docs/node-apis-helpers/
```

```suggestion
      link: /docs/node-api-helpers/
```

I think _apis_ can be singular because _helpers_ is plural.

---

I will change this last, as I need to also change page path - I will add this to TO-DO items in PR description

---

Another question - should this go below `/docs/node-apis/`? Both in sidebar hierarchy and in actual url - `/docs/node-apis/helpers`?

---

It's probably ok to keep the structure flat here? Maybe we can change it later if we want to look at restructuring the whole of the API Reference section of the docs.